src/ios/registerNativeModule.js

```diff
@@ -78,6 +81,14 @@ module.exports = function registerNativeModuleIOS(dependencyConfig, projectConfi
     });
   });
 
+  const headers = getHeadersInFolder(dependencyConfig.folder);
+  if (headers.length > 0) {
```

Comment:
we can use the same function `empty` as I proposed above:
```js
if (!emtpy(headers)) {
```
imo ^ makes code easier to read. Also not pushing this change,